frontend/src/pages/HomePage.tsx

```diff
@@ -33,116 +34,142 @@ export default function HomePage() {
-                                                image={item.image.name}
-                                                alt={item.name}
-                                            />
-                                            <CardContent sx={{flexGrow: 1}}>
-                                                <Typography gutterBottom variant="h5" component="h2">
-                                                    {item.name}
-                                                </Typography>
-                                                <Typography>
-                                                    {item.description}
-                                                </Typography>
-                                                <Typography>
-                                                    Price: {item.price}
-                                                </Typography>
-                                                <Typography>
-                                                    Category: {item.category}
-                                                </Typography>
-                                            </CardContent>
-                                            <CardActions>
-                                                <Button onClick={(event) => viewItem(event, item.id)} variant="outlined"
-                                                        size="small">View</Button>
-                                                <Button onClick={(event) => deleteItem(event, item.id)}
-                                                        variant="outlined" size="small" startIcon={<DeleteIcon/>}>
-                                                    Delete
-                                                </Button>
-                                            </CardActions>
-                                        </Card>
-                                    </Grid>
-                                ))}
-                        </Grid>
+        <ThemeProvider theme={theme}>
+            <AppBarTop/>
+            <Container component="main" maxWidth="xs">
+                <CssBaseline/>
+                <Box
+                    sx={{
+                        bgcolor: 'background.paper',
+                        pt: 8,
+                        pb: 6,
+                    }}
+                >
+                    <Container maxWidth="sm">
+                        <Typography
+                            component="h1"
+                            variant="h2"
+                            align="center"
+                            color="text.primary"
+                            gutterBottom
+                        >
+                            S.It.Co
+                        </Typography>
+                        <Typography variant="h5" align="center" color="text.secondary" paragraph>
+                            Welcome to S.It.Co - your place for collecting all your favorite items.
```

Comment:
Kleine grammatikalische Sachen: "...collecting all your favorite items." ist kleines bisschen slang, korrekterweise würde man "...collecting all *of* your favorite items." schreiben. Es geht aber auch so

Reply:
Danke dir! Ist im nächsten PR geändert. Eigentlich ist das nur ein Platzhalter, aber du weißt ja, wie das mit Platzhaltern oft ist. :D